bin/parse_samplesheet.nf

```diff
@@ -1,11 +1,11 @@
 // Function to get list of [ meta, [ fastq_1, fastq_2 ] ]
 def parse_samplesheet(LinkedHashMap row, single_end) {
```

Comment:
I'm not sure if `bin` is the best location for this function. I think the idea of the `libs/WorkflowAmpliseq` file is exactly to collect these Groovy / nextflow functions. Or won't that work as it uses parameters? In that case I would almost just add it directly into the ampliseq.nf workflow file? Just so that it is easier to locate maybe

Reply:
I'll look into it.

Reply:
I think the correct location would be either `modules/local/functions.nf` or in `subworkflows/local/parse_input.nf`, which parses inputs, including the samplesheet. I chose the latter approach, so that the code for input parsing is less scattered.